continuous_integration/environment.yml

```diff
@@ -0,0 +1,24 @@
+name: cupy
+channels:
+  - defaults
+dependencies:
+  - python=3.11
+  - cython>=3.0.0
+  - pytest>=6.2.4
+  - numpy>=1.21.1
+  - setuptools
+  - wheel
+  - pip
+  - cuda-version >= 12.6
```

Comment:
What happens if we use CUDA X.(Y+1) to compile cuda-python CUDA X.Y? (i.e. use a newer CUDA to compile an older cuda-python)?

Reply:
I don't think it's supported. As per documentation cuda-python X.Y needs to be compiled with cuda toolkit X.Y.

Reply:
So then this will fail as soon as CUDA 12.7 binaries are released.

I'm not sure how you want to keep this in line if cuda-python doesn't get released at the same time.

Reply:
@sandeepd-nv should this be `= 12.6` instead of `>=12.6`?

Reply:
Set to 12.6 exactly.